cwrc_audit_report.rb

```diff
@@ -111,7 +111,7 @@ module CWRCPerserver
     cwrc_pid = cwrc_obj['pid']
     cwrc_mtime = cwrc_obj['timestamp']
     # account for the CWRC PID ':' replaced with "_" in the Swift ID
```

Comment:
Comment could be removed. No longer the case.

Reply:
Nice catch, thx!